source/appendices/e-audit-requirements-and-tests/05-tr-book-tests.rst

```diff
@@ -0,0 +1,145 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2021 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+E.5 Title Report Audit Tests for Books and Reference_Works
+----------------------------------------------------------
+
+This section of the appendix outlines tests that MUST be run during audits for a Host_Type that is required to offer the Title Report and which includes Book and/or Reference_Work content requiring delivery of the book-related Standard Views of the Title Report (TR_B1, TR_B2, TR_B3).
+
+The Title Report will be COUNTER-compliant for book and/or reference work content if the following audit tests are passed. The book-related Standard Views of the Title Report (TR_B1, TR_B2, TR_B3) will be COUNTER-compliant if the metrics match those in the Title Report, with the appropriate aggregation.
+
+Audit test requirements vary depending on the set up of the platform, as indicated by the Options within the tests below.
+
+Title Report audit tests for books and reference works apply to Host_Types Aggregated_Full_Content, eBook, and eBook_Collection.
+
+Note that Data_Types Book and Reference_Work are included in the audit tests, as Reference_Works are included in the book-related Standard Views of the Title Report.
+
+
+E.5.1 Unique_Title_Investigations and Unique_Title_Requests
```

Comment:
I'm wondering if this separate test for the Unique_Title metrics is really necessary? They are also checked in the Access_Type test.